monitor/routerecon/recon.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func ReconForever(ctx context.Context, network netconf.Network, xprov xchain.Provider, ethCls map[uint64]ethclient.Client) {
-	if network.ID != netconf.Omega {
+	if !network.ID.IsProtected() {
```

Comment:
is mainnet routescane setup yet? we'll need to change the urls we use

Reply:
yeah that's the part I don't have context for, so wanted to check